examples/jsm/lines/LineSegmentsGeometry.js

```diff
@@ -47,7 +47,7 @@ LineSegmentsGeometry.prototype = Object.assign( Object.create( InstancedBufferGe
 
 			end.applyMatrix4( matrix );
 
-			start.data.needsUpdate = true;
+			start.needsUpdate = true;
```

Comment:
`start` is an instance of `InterleavedBufferAttribute` which has a `data` property (the instance of `InterleavedBuffer`).

Reply:
I've just noticed a strange thing: `instanceStart.clone()`, it returns a `BufferAttribute`, not a `InterleavedBufferAttribute`. So the `applyMatrix4()` method was actually correct, the issue is with cloning.

Reply:
Cloning an `InterleavedBufferAttribute` was not supported before `r117`. The method was missing so a runtime error occurred if you tried a clone. Since `r117`, `InterleavedBufferAttribute` supports now `clone()` and `toJSON()` but with de-interleaving data. This is more or less a workaround to avoid runtime errors in the editor and exporters. 

We still need to find a way to properly clone entire geometries so interleaved data are retained.

Reply:
I see, thanks for the explanation. My use case is cloning the whole geometry. I'll stick to my workaround (keeping some source data for cloning) until there's support for this. I'll close this PR then.

Reply:
Okay. Keep and eye on #19486 ;)